basic-app.Rmd

```diff
@@ -206,3 +210,184 @@ We'll come back to reactive programming multiple times, but even armed with a cu
 ## Exercises
 
 TODO
+
+<!-- 
+
+MCR: Some ideas for exercises. eval=FALSE in code chunks to prevent them from running.
```

Comment:
Done! Possible ideas for further code styling to save space:
- Present ui and server side-by-side. Especially for earlier simpler apps this would help with space-saving while maintaining ample line breaks for readability.
- Shorter line breaks for empty lines -- I like leaving an empty line between ui and server as well as between chunks of ui elements and reactive expressions. Perhaps there is a way to tweak styling of code presented to make the line breaks shorter (i.e. if this were in Word, I'd make that line a smaller font).